INTITA/Modules/ForgotPassword/ForgotPasswordViewController.swift

```diff
@@ -12,68 +12,146 @@ class ForgotPasswordViewController: UIViewController, Storyboarded {
+        
+        let registerButtonCell = UINib(nibName: "SendButtonTableViewCell", bundle: nil)
+        forgotTableView.register(registerButtonCell, forCellReuseIdentifier: "SendButtonTableViewCell")
+    }
+    
+}
 
-        emailTextField.layer.cornerRadius = 4
-        emailTextField.borderStyle = .line
-        emailTextField.keyboardAppearance = .dark
-        emailTextField.backgroundColor = .white
-        emailTextField.keyboardType = .emailAddress
-        emailTextField.placeholder = "Enter your Email"
-        emailTextField.clearButtonMode = .whileEditing
+extension ForgotPasswordViewController: UITableViewDelegate {
         
     }
+
+extension ForgotPasswordViewController: UITableViewDataSource {
     
-    func setupSendButton() {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 5
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+       
+        var cell: UITableViewCell?
         
-        let titleLocalized = "send".localized
-        sendButton.layer.cornerRadius = 8
-        sendButton.setTitle(titleLocalized, for: .normal)
+        switch indexPath.row {
+        case 0:
+            let logoCell = tableView.dequeueReusableCell(withIdentifier: "LogoForgotTableViewCell") as? LogoForgotTableViewCell
+            cell = logoCell
+        case 1:
+            let passRecoveryCell = tableView.dequeueReusableCell(withIdentifier: "PassRecoveryTableViewCell") as? PassRecoveryTableViewCell
+            cell = passRecoveryCell
+        case 2:
+            let explanationCell = tableView.dequeueReusableCell(withIdentifier: "ExplanationTableViewCell") as? ExplanationTableViewCell
+            cell = explanationCell
+        case 3:
+            let emailCell = tableView.dequeueReusableCell(withIdentifier: "EmailTableViewCell") as? EmailTableViewCell
+            emailCell?.textField.placeholder = "inputEmail".localized
+            emailCell?.textField.textContentType = .emailAddress
+            cell = emailCell
+        case 4:
+            let buttonCell = tableView.dequeueReusableCell(withIdentifier: "SendButtonTableViewCell") as? SendButtonTableViewCell
+            buttonCell?.delegate = self
+            cell = buttonCell
+        default:
+            return UITableViewCell()
+        }
+        return cell ?? UITableViewCell()
+    }
+    
+    func tableView(_ tableView: UITableView,
+                   heightForRowAt indexPath: IndexPath) -> CGFloat {
+        
+        var heightCell: CGFloat?
+        
+        switch indexPath.row {
+        case 0:
+            heightCell = widthOfView * 0.39
+        case 1:
+            heightCell = widthOfView * 0.253
+        case 2:
+            heightCell = widthOfView * 0.3
+        case 3:
+            heightCell = widthOfView * 0.264
+        case 4:
+            heightCell = widthOfView * 0.147
+        default:
+            heightCell = 0
```

Comment:
Maybe better make heightCell depending on view height, not width?

Reply:
Some elements of the screen are proportional, so in my opinion it is better to scale them depending on the width.